h/static/scripts/annotator/guest.coffee

```diff
@@ -265,6 +265,7 @@ module.exports = class Guest extends Annotator
     return confirm "You have selected a very short piece of text: only " + length + " chars. Are you sure you want to highlight this?"
 
   onSuccessfulSelection: (event, immediate) ->
+    document.getElementsByClassName('toolbar-insert-comment')[0].title = "New Note"
```

Comment:
Even if it duplicates some code I might prefer to do this in `Toolbar`. Unsure. I don't really know how to make this less ugly without refactoring the toolbar a bit. UI state is the worst.